swagger.yaml

```diff
@@ -6,7 +6,7 @@ produces:
 basePath: /
 info:
   description: Sinopia Server. Supports CRUD, following loosely LDP, on Sinopia resources. Top level requires special authorization for creating a group's graph (i.e. PCC, Stanford, etc.).
-  version: 2.0.0
+  version: 3.0.0-beta
```

Comment:
will make this 3.0.0 proper once there are tests showing that RDF resource creation and retrieval work.  if there is feedback on this PR necessitating changes, i can bump to -beta2 (i published this set of changes to npm pre-emptively, under the assumption that it'll get merged as-is, so as to allow @jgreben to use the updated version of the package more quickly).